mlflow/utils/credentials.py

```diff
@@ -41,3 +46,129 @@ def read_mlflow_creds() -> MlflowCreds:
         username=username_env or username_file,
         password=password_env or password_file,
     )
+
+
+def login(backend="databricks"):
+    """Configure MLflow server authentication and connect MLflow to tracking server.
+
+    This method provides a simple way to connect MLflow to its tracking server. Currently only
+    Databricks tracking server is supported. Users will be prompted to enter the credentials if no
+    existing Databricks profile is found, and the credentials will be saved to `~/.databrickscfg`.
```

Comment:
I think we should mention that this function calls `set_tracking_uri`.

Reply:
We can log a message after `set_tracking_uri` if we want to make it obivous that `mlflow.login` sets the tracking URI. Users don't read docs.

Reply:
Yes logging a message makes sense

Reply:
rethinking about it - now I prefer not calling `set_tracking_uri()` in this `login()` method, so we keep a consistent UX , basically users are always required to call `set_tracking_uri()`

Reply:
Does this match the FR from the Mosaic team? (I'm not sure what their current utility does)

Reply:
I like separating `set_tracking_uri` and `login`.

Reply:
@BenWilson2 @chenmoneygithub 

> Does this match the FR from the Mosaic team?

If the Mosaic team wants `mlflow.set_tracking_uri` to be called in `mlflow.login` for convenience, I'm not oppoosed to that.

Reply:
They don't ask this specifically, I will ping mosaic team after checking in this PR to get their feedback